### PR TITLE
[build] Add new flutter_app GN template for Fuchsia style package man…

### PR DESCRIPTION
**Pull request by jamesr**
…agement

### CODE REVIEW

**Comment by jamesr** [comment]
@abarth

File: build/flutter_app.gni [hunk 1]
```diff
@@ -2,37 +2,43 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
-template("flx") {
+import("//build/dart/dart_package.gni")
```
Comment (pylaligand) [inline build/flutter_app.gni]: Is there a place where to document these build template?

Comment (abarth) [inline build/flutter_app.gni]: The template declaration is a good place for that sort of documentation.

**Comment by abarth** [comment]
Looks pretty good, but please add docs to the template.

Also, I'd add an assert(is_fuchsia) to the flutter_app.gni file so folks don't try to use it in the standalone flutter build.

**Comment by abarth** [comment]
LGTM w/ those changes.

**Comment by jamesr** [comment]
Added some minimal docs to the template and ran gn format

**Comment by jamesr** [comment]
(and the assertion)

**Pull request merged**